### ISSUE TITLE
Switching to the Venmo app when logging in does not switch back, Authorize App does not appear

### ISSUE THREAD

**Issue by jhu724** [venmo/venmo-ios-sdk#73]
Steps to replicate:
- Run MiniVenmo on iPhone with Venmo app installed
- Tap login, which switches to the Venmo app
- After Touch ID login into Venmo, "authorize app" screen does not appear
- Workaround is to quit out of MiniVenmo and reopen, then everything works as expected

**Comment by eliperkins** [comment]
Hm... I'm testing this out right now and I get the `Authorize App` screen to appear consistently. Can you double check the version of the Venmo app you're testing this against and ensure that it's the latest (`6.9.1`)?

**Comment by jhu724** [comment]
Yes, I have the latest version of the Venmo app. Some more clarification:
- Force quit both MiniVenmo and Venmo (double clicking home button and swiping away)
- Open MiniVenmo and login, which brings up Venmo
- "Authorize App" does not appear
- Workaround: force quit MiniVenmo and reopen and login, "Authorize App" finally appears

**Comment by eliperkins** [comment]
Ah, yup. That does seem to be a bug on our end. I'll look into it and see what a fix will take!

**Comment by jhu724** [comment]
Thanks!

**Comment by jhu724** [comment]
Hi @eliperkins , any update on this? Thanks.

**Comment by jhu724** [comment]
Hi @eliperkins , any updates on this? I imagine this would be a critical issue for any developer using Venmo's SDK.

**Comment by jeremypchen** [comment]
Would also like to know if this is currently being looked at, @eliperkins. Thanks!
